src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/Info/HygieneRules/HygieneRulesInfoViewModel.swift

```diff
@@ -0,0 +1,54 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+struct HygieneRulesInfoViewModel {
+	
+	// MARK: - Internal
+
+	var dynamicTableViewModel: DynamicTableViewModel {
+		var model = DynamicTableViewModel([])
+		
+		model.add(
+			.section(
+			   header: .image(
+				   UIImage(imageLiteralResourceName: "Illu_Distanz"),
+				   accessibilityLabel: AppStrings.ExposureDetection.hygieneRulesTitleImageDescription,
+				   accessibilityIdentifier: AccessibilityIdentifiers.ExposureDetection.hygieneRulesTitle,
+				   height: 250
+			   ),
```

Comment:
I think we need to use the `.headlineWithImage`  like e.g. in the `FirstTicketValidationConsentViewModel` this requires the header graphic to be in the correct format as the cell already contains the gradient, let's discuss this directly :)